lesson12.md

```diff
@@ -0,0 +1,412 @@
+<!-- .slide: id="lesson12" -->
+
+# Basic Frontend - Fall 2020
+
+Lesson 12, Tuesday, 2020-10-06
+
+---
+
+### Lesson overview
+
+* recap of Javascprit data types
+* introducing Javascript objects
+* differences between objects and arrays
+* some practice
+
+---
+
+### Recap: data types
+
+What data types have we seen so far?
```

Comment:
Basic data types? Because we also learned about Array and Function

Reply:
I know, it's just that if I call those "basic data types", how should I call the others? I didn't want to add unnecessary terminology. 
So, maybe I could add "array" and "function" in the second slide, right below "What's missing?" and then I'll continue focusing on arrays. What do you think?

Reply:
IMO array/function are probably more useful to recap than undefined

Reply:
@owenlow I'm not really doing a full recap on data types. I'll mention them, but then I just focus on arrays to explain objects in relation to them

Reply:
just FIY, instead of Data Types I'll just put "Types" so that it's more technically correct